frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts

```diff
@@ -9,12 +9,13 @@ import type {
 import validate from "metabase/lib/validate";
 import type { Parameter } from "metabase-types/types/Parameter";
 import type { TemplateTag } from "metabase-types/types/Query";
+import type { Validator } from "metabase-types/forms";
 
 export const getDefaultFormSettings = (
   overrides: Partial<ActionFormSettings> = {},
 ): ActionFormSettings => ({
   name: "",
-  type: "inline",
+  type: "modal",
```

Comment:
i'm sure we will get feedback on what should be the default here, but at least given how we currently add little widgets to dashboards, I think modal makes more sense for now.